app/web/src/features/communities/discussions/DiscussionPage.tsx

```diff
@@ -85,6 +86,7 @@ export default function DiscussionPage() {
 
   return (
     <>
+      <HtmlMeta title={discussion?.title} />
```

Comment:
Looking at how `HtmlMeta` works, it's possible that the title will read "Couchers.org Beta | Couchers.org Beta" while `discussion.title` is undefined (i.e. the query is still loading) which I'm not sure we want that. We might want to set the `noSuffix` depending on the loading state instead to avoid this.

Reply:
I changed the behaviour. Now when title has default value, it will automatically omit the suffix. This should fix the problem with loading state with no extra code this component is used